src/OpenLoco/OpenLoco.cpp

```diff
@@ -1256,18 +1249,16 @@ namespace OpenLoco
 
             resetCmdline();
             registerHooks();
-            if (sub_4054B9())
```

Comment:
what did this deleted function do?

Reply:
Registered the wndclass which is handled by sdl in our code